vignettes/pr_review_guidance.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Pull Request Review Guidance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{pr_review_guidance}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
```

Comment:
"Writing Vignettes" should be followed, i.e.,
- sections should start at level 1 and the first section should be "Introduction".
- _pkgdown.yml should be updated